pages/index.tsx

```diff
@@ -7,7 +7,7 @@ import LaunchPage from "../components/LaunchPage"
 import Results from "../components/Results"
 import Chat from "../components/Chat"
 
-const GameScreen = dynamic(() => import("../components/GameScreen"))
+const GameScreen = dynamic(() => import("../components/GameScreen"), { ssr: false })
```

Comment:
why are we resetting `ssr: false`? 

is this an intentional change?

Reply:
that was my bad, this was when I was debugging for audio playback and switched to crunker